src/idl_parser.cpp

```diff
@@ -3131,7 +3131,7 @@ Offset<reflection::Field> FieldDef::Serialize(FlatBufferBuilder *builder,
 }
 
 bool FieldDef::Deserialize(Parser &parser, const reflection::Field *field) {
-  name = parser.UnqualifiedName(field->name()->str());
```

Comment:
Why was this removed?

Reply:
1. It seems to be pointless to use this function on fields, because they can't have namespaces(right?)
2. Since field doesn't have namespace, current_namespace is set to empty (by UnqualifiedName). Thus we can't get correct qualified name for nested_flatbuffer value (line 3162)

Reply:
Ah ok, makes sense.